source/docs/software/wpilib-tools/smartdashboard/verifying-smartdashboard-is-working.rst

```diff
@@ -64,7 +64,7 @@ You can verify that the robot program is generating SmartDashboard values by usi
 
 OutlineViewer is downloaded as part of the WPILib Offline Installer. For more information, see the :ref:`Windows/macOS/Linux installation guides <docs/zero-to-robot/step-2/wpilib-setup:WPILib Installation Guide>`. In Visual Studio Code, press **Ctrl + Shift + P** and type WPILib or click the WPILib logo in the top right to launch the WPILib Command Palette and select **Start Tool**, then select **OutlineViewer**.
 
-In the host box, enter your roboRIO hostname (``roboRIO-####.local`` where ``####`` is your team number with no leading zeroes). Then, click ``Start Client``.
+In the "Server Location" box, enter your team number with no leading zeroes. Then, click ``Start``.
```

Comment:
I don't think this change is correct?

Reply:
If you look at this picture and the text that was on the overview "To connect to your robot, open OutlineViewer and set the “Server Location” to be your team number. After you click start, OutlineViewer will connect."
![image](https://user-images.githubusercontent.com/15695521/101286583-e6bf4a80-37b0-11eb-8c2c-65d56cd5492e.png)

I have not personally verified this.

Reply:
I just went out to the robot and verified that if I use "6391" in the server location it works.  If I use "roboRIO-6391.local" it does not

Reply:
Yes, team number works and is more robust because it will try mdns, dns, etc. Note that the proper hostname that works is roboRIO-####-frc.local, It hasn't been roboRIO-####.local since 2015.